addons/crm/models/crm_lead.py

```diff
@@ -2282,16 +2347,14 @@ def _update_automated_probabilities(self):
             return
 
         # 1. Get all the leads to recompute created after pls_start_date that are nor won nor lost
-        # (Won : probability = 100 | Lost : probability = 0 or inactive. Here, inactive won't be returned anyway)
+        # (Won : probability = 100 and won stage | Lost : probability = 0 and inactive. Here, inactive won't be returned anyway)
         # Get also all the lead without probability --> These are the new leads. Activate auto probability on them.
         pending_lead_domain = [
-            '&',
-                '&',
-                    ('stage_id', '!=', False), ('create_date', '>=', pls_start_date),
-                '|',
-                    ('probability', '=', False),
-                    '&',
-                        ('probability', '<', 100), ('probability', '>', 0)
+            ('stage_id', '!=', False),
+            ('create_date', '>=', pls_start_date),
+            '|',
+                ('probability', '=', False),
+                ('is_won', '=', False),  # If active and P=0, we want to recompute?
```

Comment:
Here, we only recompute probas (when manually triggering the update) of Active leads, that are nor won nor lost.

However, those leads are updated in batch mode below and probabilities obtained with _pls_get_naive_bayes_probabilities(batch_mode=True)) called on them.

This is the only time batch_mode parameter is True. Then inside that method we find this:

```python
  # Get all leads values, no matter the team_id
  domain = []
  if batch_mode:
      domain = [
          '&',
              ('active', '=', True), ('id', 'in', self.ids),
              '|',
                  ('probability', '=', None),
                  '&',
                      ('probability', '<', 100), ('probability', '>', 0)
      ]
```

So two things: a bit of duplication (active = True), but maybe it is a security check?
And it seems like not recomputing 0 and 100 proba set manually is a choice. (it could happen in new heuristics: active with 0 or 100 without won stage) Do we want that?